### ISSUE TITLE
Last Route overrides previous ones

### ISSUE THREAD

**Issue by guramsutidze** [themosis/framework#600]
```
Route::get('template', [
    'tour', 
    'uses' => 'TourController@page'
]);

Route::get('template', [
    'team', 
    'uses' => 'TeamController@page'
]);
```

Last Route overrides previous ones, if conditional tag is similar

**Comment by jlambe** [comment]
Which version of the framework are you using? A fix has been added normally on 2.0-beta3 release.

**Comment by guramsutidze** [comment]
I updated framework to 2.0-beta3 and problem solved, thanks.
